lib/membrane_wav/serializer_bin/interceptor.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Membrane.WAV.SerializerBin.Interceptor do
```

Comment:
OK, seeing the implementation I think I misled you and using a bin wasn't a good idea, sorry about that. I was convinced the support for seeking in the file is needed at a parent of the serializer. While it obviously can be done inside the pipeline.

First of all, the default way to serialize to WAV should generate a valid file, this shouldn't be an optional behaviour, so I'd move the logic from Interceptor to Serializer. Then you can add a flag to turn it off - to allow using it with something different than file sink. The doc there should point to the Postprocessing module.